pumpkin-config/src/lib.rs

```diff
@@ -126,26 +128,33 @@ trait LoadConfiguration {
     where
         Self: Sized + Default + Serialize + DeserializeOwned,
     {
-        let path = Self::get_path();
+        let exe_path = env::current_exe().unwrap();
```

Comment:
In docker, the config folder is created in `/bin`, something is wrong!
It should not use current_exe as base path, instead use CWD see env::current_dir()

Reply:
Makes sense, did not saw that. Can you may test it again with the latest change